lib/project_types/node/commands/create.rb

```diff
@@ -99,7 +100,7 @@ def build(name)
         @ctx.root = File.join(@ctx.root, name)
 
         set_npm_config
-        JsDeps.install(@ctx)
+        ShopifyCli::JsDeps.install(@ctx, options.flags[:verbose] ? true : false)
```

Comment:
Isn't `options.flags[:verbose]` either `true` or `false` anyway?  If so, can't the call simply be
```
ShopifyCli::JsDeps.install(@ctx, options.flags[:verbose])
```
Or does `options.flags[:verbose]` not exist if `--verbose` isn't specified?

Reply:
It'll be `nil` if no flag is passed in since we only set the flags if the options are actually present in the call. My idea here was to just convert nil to false to ensure we're always passing in a boolean.

Reply:
Then this could just be `!options.flags[:verbose].nil?`

Reply:
It could, but it also kind of assumes this will always be a `true / nil` scenario. I'm torn. This fix is definitely better-looking but I'm not sure if the 'risk' I see is real or not.